.ci/validation.sh

```diff
@@ -4,20 +4,15 @@ set -e
 case $1 in
 
 git-diff)
-  printf "Changes to only docs/sitemap.xml:\n"
-  git diff docs/sitemap.xml
-
-  # file below is always modified on a run and updated with the current date
-  # such a change is ignored and must be verified manually instead
-  git checkout HEAD -- docs/sitemap.xml
+  git checkout HEAD
```

Comment:
I don't think the checkout is needed either. The original checkout was just to reset `sitemap.xml` when it got modified.

Reply:
Now I get it. I had read the command as checkout everything except sitemap, which was completely wrong.